packages/discord.js/typings/index.d.ts

```diff
@@ -2235,6 +2238,11 @@ export class ThreadChannel extends TextBasedChannelMixin(Channel) {
   public rateLimitPerUser: number | null;
   public type: ThreadChannelType;
   public readonly unarchivable: boolean;
+  public isPrivate(): this is this & {
+    createdTimestamp: number;
+    createdAt: Date;
```

Comment:
Should `createdAt: Date` here be prefixed with `readonly` or does that not matter?